bluepysnap/utils.py

```diff
@@ -53,6 +53,15 @@ def load_json(filepath):
         return json.load(f)
 
 
+def is_node_id(node_id):
+    """Check if node_id can be a valid node id.
+
+    Returns:
+        bool: true is node_id is a int, np.integer or CircuitNodeId. False for anything else.
+    """
+    return isinstance(node_id, (int, np.integer, CircuitNodeId))
```

Comment:
Just thinking here. If there is a dedicated method then should it also check for negative numbers? I guess should not.

Also can this function be used in https://github.com/BlueBrain/snap/blob/master/bluepysnap/nodes.py#L626 ?